clang-tools-extra/test/clang-doc/assets.cpp

```diff
@@ -0,0 +1,22 @@
+// RUN: rm -rf %t && mkdir -p %t/docs %t/build
```

Comment:
do you need `%t/build`? if not, then you can probably just use `%t` instead of `%t/docs`.

Reply:
my bad i just blindly copied from the previous test cases, yes %t would probably work better